lms/static/scripts/frontend_apps/utils/api.ts

```diff
@@ -317,6 +317,8 @@ export type PaginatedFetchResult<T> = Omit<FetchResult<T>, 'mutate'> & {
   loadNextPage: () => void;
 };
 
+export const ITEMS_PER_PAGE = 100;
```

Comment:
This assumes the page size is the same for all requests. That is currently true, but it seems like that could change since some requests will probably have a much higher cost per item than others.

Reply:
Please, ignore this PR for now. I'm planning a few changes still.